.github/workflows/build-images.yaml

```diff
@@ -25,10 +24,34 @@ jobs:
   Build:
     strategy:
       matrix:
-        example: [python]
+        example: [python, rust]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Install pnpm & protoc-gen-ts
+        uses: pnpm/action-setup@v2
+        with:
+          version: 7.12.2
+      - run: pnpm install -g protoc-gen-ts google-protobuf typescript@4.x.x 
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v2
+      - name: install rust prost plugin
+        run: |
+          cargo install protoc-gen-prost
+          cargo install protoc-gen-prost-serde
+          cargo install protoc-gen-prost-crate
+          cargo install protoc-gen-tonic
+      - name: Generate the protos
+        uses: bufbuild/buf-setup-action@v1.24.0
+      - run: buf generate
+      - run: > 
+          pip install grpcio-tools &&
+          python3 -m grpc_tools.protoc --proto_path=./proto --python_out=python --pyi_out=python --grpc_python_out=python  
```

Comment:
Can this just call build_protos.sh?

Reply:
i spent a lot of time on this... and the tldr is, we can't do it easily if we don't use buf remote registry..

the missing part is `protoc-gen-python_grpc`, which doesn't have a standalone binary(and ideally cross-platform). the only option is, build from source.

but buf has `grpc_python` built-in plugin, which generates `async` code instead. so to make it work and if we want, we can go that route...